vnext/package.json

```diff
@@ -69,5 +69,56 @@
       "minor",
       "patch"
     ]
-  }
+  },
+  "files": [
+    "/codegen",
+    "/Chakra",
+    "/Common",
+    "/DeforkingPatches",
+    "/etc",
+    "/flow",
+    "/Folly",
+    "/include",
+    "/jest",
+    "/JSI",
+    "/Libraries",
+    "!/Libraries/**/*.windesktop.*",
+    "/local-cli",
+    "/Microsoft.ReactNative",
+    "/Microsoft.ReactNative.Cxx",
+    "/Microsoft.ReactNative.Managed",
+    "/Microsoft.ReactNative.SharedManaged",
+    "/Mso",
+    "/PropertySheets",
+    "/ReactCommon",
+    "/ReactUWP",
+    "/ReactWindowsCore",
+    "/Scripts",
```

Comment:
Don't think we need `Scripts`. Those should be for development only.

Reply:
@acoates-ms: Can you confirm?

Reply:
And scripts is used.... [deploy.js#291](https://github.com/microsoft/react-native-windows/blob/master/vnext/local-cli/runWindows/utils/deploy.js#L291)

Reply:
Oh wow. `deploy.js` really shouldn't be using that. All it does is run a single command with an added title. https://github.com/microsoft/react-native-windows/blob/master/vnext/Scripts/launchPackager.bat.

We actually don't seem to use that script anywhere else, which is fun since the script actually predates the CLI using it https://github.com/microsoft/react-native-windows/pull/2302/files. Feel free to just delete the script and call the command directly if you're feeling adventurous.

Reply:
Lets file the issue and handle it as additional smaller PRs.

Reply:
#4811